storybook/package.json

```diff
@@ -1,11 +1,16 @@
 {
   "version": "0.0.0",
-  "author": "Community for NL Design System",
-  "description": "Storybook React website for the City of Amsterdam based on the NL Design System architecture",
+  "author": "Design System Team, City of Amsterdam <designsystem@amsterdam.nl>",
+  "description": "The handbook for the Amsterdam Design System, created with Storybook.",
+  "homepage": "https://designsystem.amsterdam/",
   "license": "EUPL-1.2",
   "name": "@amsterdam/storybook",
   "keywords": [
-    "nl-design-system"
+    "amsterdam",
+    "amsterdam-design-system",
+    "design-system",
+    "nl-design-system",
+    "storybook"
   ],
```

Comment:
I'm not sure how relevant this is, we don't publish this package

Reply:
The `keywords` property is required. Do you prefer an empty array over consistency with the other packages?

Reply:
> The keywords property is required.

No it isn't?

Reply:
[It is for us :)](https://github.com/Amsterdam/design-system/blob/1599936e7f647004a449083b8bfe2071d422f983/.npmpackagejsonlintrc.json#L29)

Reply:
Ah, wasn't aware of that. I don't really see why it should be required though, we can remove that rule imo. Or an empty array is fine as well, if we want to keep the rule.

Reply:
I’ve emptied it.